CMakeLists.txt

```diff
@@ -1314,6 +1316,15 @@ ELSE()
         SET_TARGET_PROPERTIES(${IMPL_TARGET} PROPERTIES PREFIX "")
     ENDIF()
 ENDIF()
+
+TARGET_INCLUDE_DIRECTORIES(${IMPL_TARGET}
+ PUBLIC
+   $<BUILD_INTERFACE:${OpenAL_SOURCE_DIR}/include>
+   $<INSTALL_INTERFACE:include>
+ PRIVATE
+   "${OpenAL_BINARY_DIR}"
+)
```

Comment:
It would probably make sense to combine this with the existing `TARGET_INCLUDE_DIRECTORIES(${IMPL_TARGET} ...` call a few lines down.

Reply:
Okay. I will do it.